react/typings/global.d.ts

```diff
@@ -23,4 +27,26 @@ interface BazaarvoiceItem {
   name: string
   price: number
   category: string
+}
+
+interface ImpressionData {
+  contentId: string
+  productId: string
+  bvProduct: string
+  contentType: string
+}
+
+interface TrackInViewData {
+  containerId: string
+}
+
+interface InViewData {
+  productId: string
+  bvProduct: string
+}
+
+//Right now this is the same of TrackInViewData but 
+//these two can receive different optional values.
```

Comment:
Add a space after `//`
```suggestion
// these two can receive different optional values.
```